games-arcade/wormux/wormux-20130927.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A free Worms clone"
+DESCRIPTION="War has been declared! Battle your favorite free software mascots in the Wormux arena. With big sticks of dynamite, grenades, baseball bats, and bazookas, you can exterminate your opponent in a 2D cartoon style scenery. The goal of the game is to destroy all of your opponents' mascots."
+HOMEPAGE="http://wormux.org"
+COPYRIGHT="<a-team>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+
+#SOURCE_URI="git://github.com/jcemelanda/wormux.git"
+srcGitRev="e0dc503460055a14a0d7fd94e66d7484264ee62b"
+SOURCE_URI="https://github.com/a-team/wormux/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="wormux-$portVersion-$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d5e66808de2e4912a64b583e3be4b8ea52cd220b8617a113507cca2c2291c6d3"
+SOURCE_DIR="wormux-$srcGitRev"
+#SOURCE_DIR="wormux.git"
+
+#ARCHITECTURES="x86_gcc2 ?x86 x86_64"
```

Comment:
```diff
 SOURCE_DIR="wormux-$srcGitRev"
-#SOURCE_DIR="wormux.git"
 
-#ARCHITECTURES="x86_gcc2 ?x86 x86_64"
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
-
 SECONDARY_ARCHITECTURES="x86"
```
Note: I haven't checked if it builds with gcc2, i.e. on x86_gcc2 primary arch.
What you can do is use `ARCHITECTURES="?x86_gcc2 ?x86 x86_64"` until you have the answer to the previous question.